### DISCUSSION TITLE
Alerts

### DISCUSSION THREAD

**Discussion by edm-ms** [Azure/avdaccelerator#63] [Ideas]
Should we consider adding alerts? 

If we could define some simple thresholds to alert on that would be important, but not too noisy it might be a good idea to include.

**Comment by danycontre** [comment]
It does sounds like a good idea, at least for the common alerts customer should pay attention too:

Most popular:
* Session host availability on host pools.
* User session disconnections (we will need to narrow this down to specific error code or sources).

Good to have:
* Host pool max user session utilization.
* Session host resource consumption.
* RTT numbers (set above a threshold).
* Session host resource consumption.
* Scaling operations if using scaling plans (VM start and deallocation).
